ArchiSteamFarm/Utilities.cs

```diff
@@ -133,6 +133,21 @@ internal static class Utilities {
 			return true;
 		}
 
+		internal static string MarkdownToText(string markdownText) {
+			if (string.IsNullOrEmpty(markdownText)) {
+				ASF.ArchiLogger.LogNullError(nameof(markdownText));
+				return null;
+			}
+
+			string text = Markdig.Markdown.ToPlainText(markdownText);
+			if (string.IsNullOrEmpty(text)) {
+				ASF.ArchiLogger.LogNullError(nameof(text));
```

Comment:
non-empty markdown can result in empty string, such as `****`, entire `if` should be removed, null != empty in this case.

Also name it either `result` or `plainText` to keep convention with input.

Reply:
Alternatively leave it like that but apply `text == null` condition instead.